packages/frameworks-express/tests/setupTests.ts

```diff
@@ -0,0 +1,23 @@
+import cryptoLib from "crypto"
+
+function setup() {
+  if (!globalThis.fetch) {
+    import("node-fetch").then((module) => {
+      // @ts-expect-error
+      globalThis.fetch = module.default
+      // @ts-expect-error
+      globalThis.Request = module.Request
+      // @ts-expect-error
+      globalThis.Response = module.Response
+      // @ts-expect-error
+      globalThis.Headers = module.Headers
+    })
+  }
+
+  if (!globalThis.crypto) {
+    // @ts-expect-error
+    globalThis.crypto = cryptoLib.webcrypto
+  }
+}
+
+setup()
```

Comment:
Neat one-liners:

```suggestion
globalThis.fetch ??= require("undici").fetch
globalThis.crypto ??= require("node:crypto").webcrypto
```

This would require `undici` to be added as a dev dependency, which is fine.